### PR TITLE
Update quickstart matrix.

### PR DESCRIPTION
**Pull request by faxu**
Add Java

### CODE REVIEW

**Comment by prasanthpul** [comment]
How about Android?

File: index.html [hunk 1]
```diff
@@ -192,7 +194,7 @@ <h3 id="selectHardwareAcceleration">Hardware Acceleration</h3>
 	                                    <div class="col-md-9 r-content pr-0 pl-md-4" role="listbox" id="listbox-4" aria-labelledby="selectHardwareAcceleration" aria-describedby="decriptionHardwareAcceleration">
 	                                        <div class="row hardwareAcceleration">
 	                                            <div class="col-lg-2dot5 col r-option version selected" role="option" tabindex="0" aria-selected="true" id="DefaultCPU">
-	                                                <span>Default <abbr>CPU</abbr></span></div>
+	                                                <span>Default&nbsp; <abbr>CPU</abbr></span></div>
```
Comment (prasanthpul) [inline index.html]: is this change intentional?

Comment (faxu) [reply index.html]: > is this change intentional?
yes, looks a bit cleaner to separate the words "Default CPU" vs DefaultCPU since it's a conceptual term

**Comment by faxu** [comment]
> How about Android?

will consider adding it later after some open issues are sorted out.
